src/stubs/LocalizedVagrantfile

```diff
@@ -0,0 +1,23 @@
+require 'json'
+require 'yaml'
+
+VAGRANTFILE_API_VERSION = "2"
+confDir = $confDir ||= File.expand_path("~/.composer/vendor/laravel/homestead")
```

Comment:
But it seems like just by changing that line to use /vendor/laravel/homestead you could eliminate the need to have Homestead globally installed.